src/main/java/com/solvd/review/service/impl/ReviewServiceImpl.java

```diff
@@ -1,27 +1,53 @@
 package com.solvd.review.service.impl;
 
 import com.solvd.review.domain.Review;
+import com.solvd.review.domain.criteria.SearchCriteria;
+import com.solvd.review.domain.exception.ResourceNotFoundException;
 import com.solvd.review.persistence.ReviewRepository;
 import com.solvd.review.service.ReviewService;
+import com.solvd.review.service.property.MicroserviceProperty;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
-import java.util.List;
+import java.time.LocalDate;
+import java.util.Objects;
 
+@Slf4j
 @Service
 @RequiredArgsConstructor
 public class ReviewServiceImpl implements ReviewService {
 
     private final ReviewRepository reviewRepository;
+    private final WebClient.Builder webClientBuilder;
+    private final MicroserviceProperty microserviceProperty;
```

Comment:
@hizmailovich 
what is the microservice property?
rename it, and the code will be more understandable and maintainable by others